sdk/personalizer/Azure.AI.Personalizer/api/Azure.AI.Personalizer.netstandard2.0.cs

```diff
@@ -1,5 +1,73 @@
 namespace Azure.AI.Personalizer
 {
+    public partial class DecisionContext
+    {
+        public DecisionContext() { }
+        public DecisionContext(Azure.AI.Personalizer.PersonalizerRankMultiSlotOptions rankRequest, System.Collections.Generic.Dictionary<string, System.Collections.Generic.IList<object>> slotIdToFeatures) { }
+        public DecisionContext(System.Collections.Generic.IEnumerable<object> contextFeatures, System.Collections.Generic.List<Azure.AI.Personalizer.PersonalizerRankableAction> rankableActions) { }
+        [System.Text.Json.Serialization.JsonIgnoreAttribute(Condition=System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull)]
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("FromUrl")]
+        public System.Collections.Generic.List<string> ContextFeatures { get { throw null; } }
+        [System.Text.Json.Serialization.JsonPropertyNameAttribute("_multi")]
+        public Azure.AI.Personalizer.DecisionContextDocument[] Documents { get { throw null; } set { } }
```

Comment:
Do `Documents` and `Slots` need to be arrays. Would it not be easier for setters if they were IList<T> or another generic enumerable type? Unless there is a good reason not to, that is [the recommendation](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-return-model-collections).

Reply:
sure.